app/views/pages/get-started.liquid

```diff
@@ -59,7 +59,8 @@ npm install -g @platformos/pos-cli
 4. Create an empty folder for your project
 5. Navigate to the project's folder in the command line
   <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
-cd x:\platformos-hello
+mkdir platformos-hello
```

Comment:
If you want to introduce a command then the 4th point is not needed and you should explain what is it doing.

It was intentional that I did not used it as this is easier to understand. By introducing mkdir you are not lovering the bar for entry level people.

Reply:
I'll move that part to the previous step in my next commit, but the main problem here is the windows syntax: `cd x:\` -- this doesn't really make sense on other platforms.
> lowering the bar for entry level people
I don't think that `mkdir` sets the bar too high for software developers, but if you think so I can remove the whole step.

Reply:
I'd love to be able to follow a tutorial and expect any command in it to work [ assuming they are in the correct order ]. I think that the red error `cd : Cannot find drive. A drive with the name 'x' does not exist.` wouldn't help much :) so I'd love to keep `mkdir platformos-hello` followed by `cd platformos-hello`